gen-ai/orchestrator-server/src/main/python/server/src/gen_ai_orchestrator/routers/em_providers_router.py

```diff
@@ -126,6 +128,12 @@ async def get_em_provider_setting_by_id(
             api_base='https://doc.tock.ai/tock',
             api_version='2023-05-15',
         )
+    elif provider_id == EMProvider.BLOOMZ:
+        return BloomzEMSetting(
+            provider=EMProvider.BLOOMZ,
+            api_base='https://bloomz-3b-retriever-v2.l00001208-e99-354.rec.cl1.arkea.com/docs',
```

Comment:
plutôt mettre un api_base d'exemple, plus open source, car celui ci est propre à Arkea
```suggestion
            api_base='https://doc.tock.ai/tock',
```